arcgis-ios-sdk-samples/Layers/Edit KML ground overlay/EditKMLGroundOverlayViewController.swift

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class EditKMLGroundOverlayViewController: UIViewController {
+    // MARK: Storyboard views and properties
+    @IBOutlet var sceneView: AGSSceneView! {
+        didSet {
+            sceneView.scene = makeScene(groundOverlay: overlay)
+            // Move the viewpoint to the ground overlay.
+            let overlayCenter = overlay.geometry.extent.center
+            let camera = AGSCamera(lookAt: overlayCenter, distance: 1250, heading: 45, pitch: 60, roll: 0)
+            let targetExtent = overlay.geometry as! AGSEnvelope
+            sceneView.setViewpoint(AGSViewpoint(targetExtent: targetExtent, camera: camera))
```

Comment:
If you know that the overlay's geometry is an envelope (as evidenced by the force cast to `AGSEnvelope`), then there's no need to get the extent of the geometry. This would be equivalent, no?

```suggestion
            let targetExtent = overlay.geometry as! AGSEnvelope
            let camera = AGSCamera(lookAt: targetExtent.center, distance: 1250, heading: 45, pitch: 60, roll: 0)
            sceneView.setViewpoint(AGSViewpoint(targetExtent: targetExtent, camera: camera))
```